package.json

```diff
@@ -47,16 +47,17 @@
 		"@prismicio/mock": "0.0.10",
 		"@prismicio/types": "^0.1.27",
 		"@rollup/plugin-inject": "^4.0.4",
-		"@sveltejs/kit": "1.0.0-next.328",
-		"@typescript-eslint/eslint-plugin": "^5.23.0",
-		"@typescript-eslint/parser": "^5.23.0",
+		"@sveltejs/kit": "^1.0.0-next.587",
+		"@sveltejs/package": "^1.0.0-next.6",
+		"@typescript-eslint/eslint-plugin": "^5.46.1",
+		"@typescript-eslint/parser": "^5.46.1",
 		"c8": "^7.11.2",
 		"eslint": "^8.15.0",
 		"eslint-config-prettier": "^8.5.0",
-		"eslint-plugin-prettier": "^4.0.0",
+		"eslint-plugin-prettier": "^4.2.0",
 		"eslint-plugin-svelte3": "^4.0.0",
 		"eslint-plugin-tsdoc": "^0.2.16",
-		"prettier": "^2.6.2",
+		"prettier": "^2.8",
```

Comment:
`^2.8` is technically valid, but is inconsistent compared to other dependency version numbers. Could we use the full version number?

Reply:
👍